api-scripts/loan-storage/README.md

```diff
@@ -0,0 +1,18 @@
+## Workflow steps:
+
+##### api for GET_loan-storage_loans.jmx
+1. GET /loan-storage/loans?query=(userId==${user_id} and status.name<>Closed)
+
+
+
+## Modules required to be enabled as a pre-requisite to run JMeter script:
+##### Backend:
+- mod-circulation-18.0.9
+- mod-circulation-storage-11.0.0
+- mod-authtoken-2.4.0
+- mod-permissions-5.9.0
+- okapi-2.38.0
+##### Frontend:
+- folio_circulation-2.0.0
+
+- FOLIO release: HONEYSOUCKLE
```

Comment:
Please add a comment on how to run the test, specifically where the user IDs are obtained from?
1) Restore the database
2) Run the circ data load script
3) anything else..?